### PR TITLE
Mime fixes

### PR DESCRIPTION
**Pull request by dgershman**

### CODE REVIEW

**Comment by dgershman** [comment]
@pjaudiomv can you review?

**Review by pjaudiomv — APPROVED** [review]
looks good to me, my only comment is sometimes I see a `return null` and other times just a `return`. Is there any reason for this.

**Comment by dgershman** [comment]
Specifically where?

File: legacy/_includes/functions.php [hunk 1]
```diff
@@ -2042,7 +2042,7 @@ function getIvrResponse($redirected_from = null, $prior_digit = null, $expected_
                 <Redirect><?php echo $redirected_from . $qs?></Redirect>
             </Response>
             <?php
-            exit();
+            return null;
```
Comment (pjaudiomv) [inline legacy/_includes/functions.php:2045]: here we return null

File: legacy/address-lookup.php [hunk 2]
```diff
@@ -15,7 +14,7 @@
         <Redirect method="GET">input-method.php?Digits=<?php echo $_REQUEST["SearchType"] . "&amp;Retry=1"; ?></Redirect>
     </Response>
      <?php
-        exit();
+        return;
```
Comment (pjaudiomv) [inline legacy/address-lookup.php:17]: here we just return

Comment (dgershman) [reply legacy/address-lookup.php:17]: this is an inline php return

File: legacy/_includes/functions.php (hunk 1 above)
Comment (dgershman) [reply legacy/_includes/functions.php:2045]: this is a function that returns null

File: legacy/address-lookup.php (hunk 2 above)
Comment (pjaudiomv) [reply legacy/address-lookup.php:17]: ugh duh, I should of looked up more. thanks. :|

**Pull request merged**